application/src/main/kotlin/net/corda/v5/application/marshalling/json/JsonWriter.kt

```diff
@@ -401,6 +398,13 @@ interface JsonWriter {
      * data is intended to be a field or value, it does not add any separators or escaping, nor does it require that the
      * Json output that results is valid Json.
      *
```

Comment:
Documentation tweaks after I saw the Json output in tests to make it clearer to the end user what the result of certain actions are.